allPost2.php

```diff
@@ -0,0 +1,35 @@
+<?php
+    include('include/init.php');
+    function passImage($image) {
+        echo $image;
+    }
```

Comment:
Don't think is being used right now? You can keep it if you have plans to use it later, but if it's a remnant I would clear it out.